### DISCUSSION TITLE
What kind of interfaces are exported with the Wasi version? Is it the full C interface?

### DISCUSSION THREAD

**Discussion by DamianReeves** [quickjs-ng/quickjs#419] [Q&A]
What kind of interfaces are exported with the Wasi version? Is it the full C interface?
How can I extend quickjs with the Wasi build, or is that out of scope for WASI?

**Comment by saghul** [comment]
Hey there! At the moment the published wasm module is the entire interpreter, so when you run it you get the REPL, or you can launch scripts.

**Reply by DamianReeves** [reply]
Can it be used as a library where I use the exports to call into the C API, or is it a start only WASM module.

**Reply by saghul** [reply]
You got me there, not sure, I think the build we produce is start only.
